src/code/code_n64dd_800AD410.c

```diff
@@ -11,8 +11,8 @@ vu8 D_80121214;
 
 void func_800AD410(void) {
     if (!D_80121210) {
-        DmaMgr_RequestSync(_n64ddSegmentStart, (uintptr_t)_n64ddSegmentRomStart,
-                           _n64ddSegmentRomEnd - _n64ddSegmentRomStart);
+        DMA_REQUEST_SYNC(_n64ddSegmentStart, (uintptr_t)_n64ddSegmentRomStart,
+                         _n64ddSegmentRomEnd - _n64ddSegmentRomStart, "../code_n64dd_800AD410.c", 0);
```

Comment:
I think I'd be more comfortable with something like
```suggestion
                         _n64ddSegmentRomEnd - _n64ddSegmentRomStart, UNK_FILE_NAME, UNK_LINE_NUMBER);
```
just so it's completely unambiguous to any reader who isn't in the loop that these are fake, in that we can't know what they originally were. These macros could then expand to `__FILE__, __LINE__` or just to `__FILE__, 0`

Reply:
Alright, I've created `UNK_FILE` (and turns out `UNK_LINE` already exists). I don't think the macros should expand to `__FILE__` and `__LINE__`, it seems weird to mix in decomp filenames and line numbers in with the originals. Instead I think it would be better to have versions of the debug macros that put in the real `__FILE__` and `__LINE__` everywhere if you wanted this for debugging (as @Dragorn421 has suggested before)